app/views/govuk_publishing_components/components/_save_this_page.html.erb

```diff
@@ -0,0 +1,24 @@
+<%
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
+  signed_in ||= false
+  page_is_saved ||= false
+  page_path ||= nil
+  form_action ||= page_is_saved ? "/my-fancy-delete-action" : "/my-fancy-save-action"
```

Comment:
Same as the other comment: it would be good to explicitly include the word `placeholder` to avoid confusion.